docs/api.md

```diff
@@ -345,6 +348,30 @@ console.log(await page.evaluate('location.href'));
 
 > **NOTE** Use [`page.waitForLoadState([state[, options]])`](#pagewaitforloadstatestate-options) to wait until the page gets to a particular state (you should not need it in most cases).
 
+#### browserContext._disableScreencast()
+- returns: <[Promise]>
+**experimental**
+Disables automatic screencast for new pages. Already started screencasts won't be
+interrupted.
+
+#### browserContext._enableScreencast(options, callback)
```

Comment:
I think we should treat these as downloads:
- LaunchOptionsBase.videosPath: string
- BrowserContextOptions.recordVideos: { width: number,  height: number }| undefined

Reply:
Done.